panther_analysis_tool/core/parse.py

```diff
@@ -63,6 +63,29 @@ def parse_filter(str_filters: Optional[List[str]]) -> Tuple[List[Filter], List[F
     return filters, filters_inverted
 
 
+def get_filters(str_filters: Optional[List[str]]) -> Tuple[List[Filter], List[Filter]]:
+    filters, filters_inverted = parse_filter(str_filters)
+    filters, filters_inverted = add_status_filters(filters, filters_inverted)
+    return filters, filters_inverted
```

Comment:
This name seems like it does the same thing as `parse_filters`. Which confused me at first. I would add `status` to the name to clarify. Like `get_filters_with_status_filters` or something.

Reply:
updated in [747572c](https://github.com/panther-labs/panther_analysis_tool/pull/680/commits/747572c7280e3ef5deaf6e111fa9d4a8feb67c48)
and [45d7745](https://github.com/panther-labs/panther_analysis_tool/pull/680/commits/45d7745ec481f01b4105860b73933119c36b6da5)